printable.module

```diff
@@ -156,29 +160,34 @@ function printable_entity_view(array &$build, EntityInterface $entity, EntityVie
   $printable_manager = \Drupal::service('printable.entity_manager');
   $link_builder = \Drupal::service('printable.link_builder');
   if ($printable_manager->isPrintableEntity($entity)) {
-    $setting = \Drupal::service('config.factory')->get('printable.settings')->get('printable_print_link_locations');
+    $print_setting = \Drupal::service('config.factory')->get('printable.settings')->get('printable_print_link_locations');
+    $pdf_setting = \Drupal::service('config.factory')->get('printable.settings')->get('printable_pdf_link_locations');
     // Get the name of entity over here.
```

Comment:
Just like in template_preprocess_printable() use ```$config = \Drupal::config('printable.settings');``` it makes the code more readable and probably a little faster.